conn.go

```diff
@@ -1061,6 +1096,13 @@ func (c *Conn) serverHandshake() error {
 			return err
 		}
 	}
+	if sendALPN {
```

Comment:
Why can't you just add the extension above or at least use serverALPN as the conditional